library/src/main/java/com/bumptech/glide/load/engine/Engine.java

```diff
@@ -293,6 +293,10 @@ public void onResourceReleased(Key cacheKey, EngineResource resource) {
         }
     }
 
+    public void clearDiskCache() {
+        diskCacheProvider.getDiskCache().clear();
```

Comment:
I found a use case: [ApplicationManifest.xml > application > @manageSpaceActivity](http://developer.android.com/guide/topics/manifest/application-element.html#logo), this attribute replaces the "Clear Data" button with a "Manage Storage" button which starts the above activity. An example I could think of is if a user needs space and sees that the app is huge, pressing "Clear Data" would wipe out all personalization/data. I know there's a "Clear Cache", but a lot of apps don't put their volatile data there for some reason. Anyway, I would put a "Clear image cache" button in my storage manager to allow freeing up space. Weirdly the [API for "Clear Data"](http://developer.android.com/reference/android/app/ActivityManager.html#clearApplicationUserData()) button was added in API 19, while the [attribute existed since API 1](http://developer.android.com/reference/android/content/pm/ApplicationInfo.html#manageSpaceActivityName)...